tests/_data/manifests/src/integrator-manifests-master/TestIntegratorGlossary/1.0.3/installer-manifest.json

```diff
@@ -1,11 +1,11 @@
 {
     "glossary-key": {
         "new": {
-            "shopping_list.pre.check.product_merchant_inactive_test": {
+            "shopping_list.pre.check.product_merchant_inactive_new": {
                 "en_US": "Merchant is inactive.",
                 "de_DE": "Der Händler ist nicht aktiv."
             },
-            "shopping_list.pre.check.product_merchant_not_approved_test": {
+            "shopping_list.pre.check.product_merchant_not_approved_new": {
```

Comment:
Where is the existing key added here that also exists in projects
You renamed this wrongly in the project file, or need to use `_existing_already` or alike here.